bin/adhoc-scripts/parseUnifiedCampaigns.py

```diff
@@ -72,6 +78,9 @@ def __init__(self):
         self.parser.add_argument("--fout", action="store",
                                  dest="fout", default="",
                                  help="Output json file with all the campaigns created")
+        self.parser.add_argument("--cin", action="store",
```

Comment:
I doubt that you need extra parameter for file input. Instead (see below), I suggest to add parameter which will specify action, e.g. add-test-schemas.